frontend/src/util/timer.js

```diff
@@ -20,16 +20,17 @@ export const Timer = ({
 
         // timer finished
         if (time === duration) {
-            onFinish && onFinish();
-            return;
+            return onFinish ? onFinish() : void 0;
         }
 
         // callback after interval
         if (time - lastTime > interval) {
             lastTime = time;
 
             // tick
-            onTick && onTick(time, delta);
+            if (onTick) {
+                onTick(time, delta);
+            }
```

Comment:
I thought that above was shorthand for below?

Reply:
It is! Reverted.